src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -110,6 +110,67 @@ internal byte[] ReadNextEncodedValue()
             return encodedValue;
         }
 
+        internal byte[] ReadRemainingData()
+        {
+            int remainingBytes = _end - _position;
```

Comment:
The _position is always less or equal to _end, I can sanitize it though, sounds like a good idea. The case when it's 0 is possible, don't know how likely it is, but I'll make sure it returns the static instance.